*: change "synchronize" to "replicate" for migration related description (#1203)
pingcap/docs: dev/how-to/deploy/data-migration-with-ansible.md

## Step 8: Edit variables in the `inventory.ini` file

> This step shows how to edit the variable of the deployment directory, how to configure the relay log replication position and the relay log GTID replication mode, and explains the global variables in the `inventory.ini` file.

Reviewer: We would normally just say "relay log position". I was looking to provide a suggestion on how to improve it, but it is quite repetitive with the text below. Maybe for this initial summary just say this:
```suggestion
This step shows how to make configuration changes to the `inventory.ini` file.
```

    dm-master ansible_host=172.16.10.71 deploy_dir=/data1/deploy
    ```

> ### Configure the relay log replication position

Reviewer: ```suggestion
### Configure the relay log position
```

> **Note:**
>

> > If `relay_binlog_name` is not set, DM-worker pulls the binlog starting from the earliest existing binlog file of the upstream MySQL or MariaDB. In this case, it can take a long period of time to pull the latest binlog for the data replication task.

Reviewer: It is a bit ambiguous here, it is not going to take a long time to pull the latest file. It is going to retrieve all the files and that is why it takes so long.
```suggestion
> If `relay_binlog_name` is not set, DM-worker pulls the binlog starting from the earliest existing binlog file of the upstream MySQL or MariaDB. In this event, it may take a significant amount of time to retrieve all of the binlog files.
```

### Enable the relay log GTID replication mode

In a DM cluster, the relay log processing unit of DM-worker communicates with the upstream MySQL or MariaDB to pull its binlog to the local file system.

You can enable the relay log GTID replication mode by configuring the following items. Currently, DM supports MySQL GTID and MariaDB GTID.

> - `enable_gtid`: to enable the relay log GTID replication mode to deal with scenarios like master-slave switch

Reviewer: ```suggestion
- `enable_gtid`: to enable GTID mode. This helps improve the handling of replication topology changes, such as a switch between master and slave.
```
